apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -45,6 +45,36 @@ function render_fallback_coming_soon_page() {
 	include __DIR__ . '/fallback-coming-soon-page.php';
 }
 
+/**
+ * This filter makes sure it's possible to fetch `wpcom_public_coming_soon` option via public-api.wordpress.com.
+ *
+ * @param object $options array Retrieved site options, ready to be returned as API respomnse.
+ *
+ * @return array current value of `wpcom_public_coming_soon`
+ */
+function add_public_coming_soon_to_settings_endpoint_get( $options ) {
+	$options['wpcom_public_coming_soon'] = (int) get_option( 'wpcom_public_coming_soon' );
+
+	return $options;
+}
+add_filter( 'site_settings_endpoint_get', __NAMESPACE__ . '\add_public_coming_soon_to_settings_endpoint_get' );
```

Comment:
@adamziel 

We're adding this filter like you did in D37770-code. 

Any reason not to do it in the Editing Toolkit?

Reply:
Because of all the async/distributed stuff, It took some days of drawing to figure out the wpcomsh parts - I'm not familiar with editing toolkit enough to be able to answer your question @ramonjd :(